File: prime.cpp
#include<iostream>
using namespace std;
class Prime
{
    int num,i;
    public : void display()
    {
        cout<<"enter the number:";
        cin>>num;
        for(i=2;i<num;i++)
        {
            if(num%i==0)
            {
                cout<<"it is not a prime number";break;
            }
        else
        {
            cout<<"it is a prime number";break;
        }
        }
    }
};
int main()
{
    Prime p1;
    p1.display();
}

File: JAVAAssignment 3/Prime.java
import java.util.Scanner;
class Prime
{
public static void main(String[] args)
{
int num,i;
System.out.println("Enter the number");
Scanner sc=new Scanner(System.in);
num=sc.nextInt();
for(i=2;i<num;i++)
{
if(num%i==0)
{
System.out.println("It is not a prime number");break;
}
else
{
System.out.println("It is a prime number");break;
}
}
}
}
